wavefront/server/modules/agents_module/agents_module/controllers/workflow_controller.py

```
@@ -126,60 +123,49 @@ async def generate_inference_stream():
-                    'timestamp': asyncio.get_event_loop().time(),
+                    'timestamp': asyncio.get_running_loop().time(),
                 }
                 yield f'data: {json.dumps(output_event)}\n\n'
-                await asyncio.sleep(0.1)  # remove it later
 
                 logger.info(
-                    f'Streaming inference completed for user {user_id}, workflow {namespace}/{workflow_id}'
+                    f'Streaming inference completed for execution {execution_id}, workflow {namespace}/{workflow_id}'
                 )
 
             except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)